src/_pytest/fixtures.py

```diff
@@ -1536,6 +1536,8 @@ def _getautousenames(self, node: nodes.Node) -> Iterator[str]:
     def _getusefixturesnames(self, node: nodes.Item) -> Iterator[str]:
         """Return the names of usefixtures fixtures applicable to node."""
         for mark in node.iter_markers(name="usefixtures"):
+            if not mark.args:
+                warnings.warn(f"Warning: empty usefixtures in {node.name}.")
```

Comment:
Instead of using a direct warning, inter_marks_with_nodes ought to be used and node.warn should be used to ensure the warning reports the definition site for warning about the marks

Reply:
I'm sorry, I don't understand what you mean by inter_marks_with_nodes. I was able to switch to node.warn, but I don't see inter_marks_with_nodes in the documentation. Could you please clarify?

Reply:
My bad,the method name was incorrect 

https://docs.pytest.org/en/8.1.x/reference/reference.html#pytest.nodes.Node.iter_markers_with_node

Reply:
Oh okay, understood. Thanks!